app/controllers/application_controller/ci_processing.rb

```diff
@@ -44,6 +44,13 @@ def set_ownership
     if @explorer
       @sb[:explorer] = true
       ownership(ownership_items)
+      if @ownershipitems.empty?
+        add_flash(_('None of the selected items ownership can be changed'), :error)
```

Comment:
Perhaps something like this -
`Ownership can not be changed for any of the selected items`
or -
`None of the selected items allow ownership changes`
may be easier to understand.